src/components/donation/Steps/Submit/Crypto/Crypto.tsx

```diff
@@ -0,0 +1,128 @@
+import { useState } from "react";
+import { EstimateStatus, isSuccess } from "./types";
+import { TxPackage } from "types/tx";
+import { ConnectedWallet, isCosmos, isEVM, isTerra } from "types/wallet";
+import { isDisconnected, useWalletContext } from "contexts/WalletContext";
+import { Info, LoadingStatus } from "components/Status";
+import { CryptoSubmitStep } from "slices/donation";
+import { chains } from "constants/chains";
+import Breakdown from "./Breakdown";
+import Container from "./Container";
+import WalletSelection from "./WalletSelection";
+
+export default function Crypto(props: CryptoSubmitStep) {
```

Comment:
Should align the button that navigates back to the profile page:
Crypto donation - "Cancel + Continue" buttons:
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/66b88326-a500-46c6-b70a-f09b9999b9ad)

PayPay & Stripe - "Back + Continue" buttons:
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/1ce3d26d-0973-4a3a-b4bd-5cbe834e8f6f)

DAF - no buttons to navigate back:
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/6d65dfbc-cd16-483c-97aa-a79775d2d4a9)

Stocks - no buttons to navigate back:
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/30e316d9-3ebb-48f2-bb4e-41e5296bdcaa)

-------

1. Either all buttons are named "Back" or all are named "Cancel"
2. Either all donation methods should have the "Back" button or none should (user can still go back using by clicking the built-in back button in their browser/mobile phone OR by using breadcrumbs)
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/5b1abc50-cfe4-4723-897d-26ecd66e907b)

Reply:
@misicnenad, I believe "cancel" was the one used on prev design as this is the first part of the multistep, so no prev step to go "back" to - switching all to "cancel" for now for consistency. Ultimately, all would be transferred a top level "cancel" (fixes DAF and Stocks concern) if its the intention - documented [here](https://linear.app/better-giving/issue/BG-1104/move-cancel-button-at-the-top-of-the-page)

Reply:
💯